thoth-app/src/models/mod.rs

```diff
@@ -17,8 +17,9 @@ macro_rules! graphql_query_builder {
         use yewtil::fetch::MethodBody;
 
         use $crate::THOTH_GRAPHQL_API;
-
+        #[allow(dead_code)]
```

Comment:
Why are these needed?

Reply:
Because everything in the view is coming from `Publication`, rather than `Location`, these are never used for `Location` However, `FetchLocation` and  `FetchActionLocation` have to stay in `thoth-app/src/models/location/location_query.rs` to conform to the structure in `macro_rules! graphql_query_builder`. 

So if I don't use `#[allow(dead_code)]`, I get an error from `cargo clippy --all --all-targets --all-features -- -D warnings`:

```
error: type alias `FetchLocation` is never used
    --> thoth-app/src/models/location/location_query.rs:30:5
```

Hope this makes sense!

Reply:
Is this still needed now that the `location_query` module has been removed?

Reply:
no, it doesn't seem they are -- good catch!